src/api/python/cvc5.pxi

```diff
@@ -533,6 +630,20 @@ cdef class Result:
 
 
 cdef class RoundingMode:
+    """
```

Comment:
This may be the wrong place, but: can we make `RoundingMode` a python enum?

Reply:
Good question. I see @makaimann translated this to python. @makaimann, any reason this was not a python `enum`?